stubs/caldav/caldav/lib/error.pyi

```diff
@@ -19,6 +19,6 @@ class PutError(DAVError): ...
 class DeleteError(DAVError): ...
 class NotFoundError(DAVError): ...
 class ConsistencyError(DAVError): ...
-class ReponseError(DAVError): ...
+class ResponseError(DAVError): ...
```

Comment:
Similarly, this really is called `ReponseError` in caldav. I think it's worth adding a comment here in case a future reader is confused.